tck-test/client.cpp

```diff
@@ -37,6 +37,8 @@ int main(int argc, char* argv[]) {
   folly::SocketAddress address;
   address.setFromHostPort(FLAGS_ip, FLAGS_port);
 
+  LOG(INFO) << "Creating client to connect to " << address.describe();
```

Comment:
`operator<<(std::ostream&, const folly::SocketAddress&)` already exists I think, no need for `.describe()`.